plugins/dell-dock/dell-dock.quirk

```diff
@@ -126,15 +129,32 @@ DellDockHubVersionLowest = 1.31
 DellDockBlobMajorOffset = 0x400a
 DellDockBlobMinorOffset = 0x4009
 
-# USB4 controller
+# USB4 controller (TBT)
 [TBT-00d4b071]
-Name = USB4 controller in Dell dock
+Name = USB4 controller (TBT) in Dell dock
 Summary = USB4 controller
 Vendor = Dell Inc.
 VendorId = TBT:0x00D4
 ParentGuid = USB\VID_413C&PID_B06E&hub&embedded
 FirmwareSizeMin=0x40000
 FirmwareSizeMax=0x80000
-Flags = skips-restart,require-ac,dual-image
+Flags = skips-restart,require-ac,dual-image,updatable
 Icon = thunderbolt
 InstallDuration = 46
+
+# USB4 controller (USB)
+[USB\VID_8087&PID_0B40]
```

Comment:
So I think you're doing this a bit wrong in terms of duplication.  
1. This key shows up twice.
2. After vid/pid match (from unprobed key above) then you should add a guid for Dell's instance via `TBT-00d4b071` and then re-use the existing entry from your last commit.